packages/mui-material-next/src/styles/sxConfig.ts

```diff
@@ -33,6 +40,23 @@ function createPaletteStyle(options: PaletteStyleOptions = { prop: 'color' }) {
   return fn;
 }
 
+// eslint-disable-next-line no-restricted-globals
+const isNumber = (value: string | number) => typeof value === 'number' || !isNaN(parseFloat(value));
+
+const borderRadius = (props: Record<string, any>) => {
```

Comment:
cc @siriwatknp the `borderRadius` sx key was also updated. Your proposed way for solving the conflicts works in this case too :)

Reply:
Nice.